annotate.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
 
   s.specification_version = 4 if s.respond_to? :specification_version
   s.add_runtime_dependency(%q<rake>, '>= 10.4', '< 14.0')
-  s.add_runtime_dependency(%q<activerecord>, ['>= 3.2', '< 7.0'])
+  s.add_runtime_dependency(%q<activerecord>, ['>= 3.2', '< 7.1'])
```

Comment:
Wouldn't it be better to use `8.0` instead? I think it will be using the same rationale for the existing file: by relying on the fact that a major Rails version will not be significantly incompatible with this gem.

Reply:
See https://github.com/ctran/annotate_models/pull/915

Reply:
Done